flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -112,8 +111,10 @@ public Builder project(TableSchema schema) {
       return this;
     }
 
-    public Builder limit(long newLimit) {
-      contextBuilder.limit(newLimit);
+    public Builder limit(Long newLimit) {
```

Comment:
why changing this?

Reply:
This was a mistake in my last PR job. `limit` was left out. I'll fix it here. It should be passed into options instead of calling method `contextBuilder.limit(newLimit)` directly.